pandas/core/indexes/base.py

```diff
@@ -1340,17 +1345,40 @@ def set_names(self, names, level=None, inplace=False):
     def rename(self, name, inplace=False):
         """
         Set new names on index. Defaults to returning new index.
+        Length of names must match number of levels in MultiIndex.
```

Comment:
Leave a blank line between the two lines above (the first line of the docstring should be a single line, see http://pandas.pydata.org/pandas-docs/stable/contributing_docstring.html#docstring for details)

Reply:
I changed some line with your suggestion. 
and I put `\` in the MultiIndex example to avoid error. and what happened in validation error.. 
thank you so much.

```python
################################################################################
####################### Docstring (pandas.Index.rename)  #######################
################################################################################

Set new names on index. Defaults to returning new index.

Length of names must match number of levels in MultiIndex.

Parameters
----------
name : str or sequence
    name(s) to set
inplace : bool
    if True, mutates in place

Returns
-------
new index (of same type and class...etc) [if inplace, returns None]

Examples
--------
>>> idx = pd.Index([1, 2, 3, 4], name='foo')
>>> idx.rename('bar')
Int64Index([1, 2, 3, 4], dtype='int64', name='bar')

>>> idx = pd.MultiIndex.from_tuples([(1, u'one'), (1, u'two'),                                            (2, u'one'), (2, u'two')],                                            names=['foo', 'bar'])
>>> idx.rename(['bar', None], inplace=True)
>>> idx
MultiIndex(levels=[[1, 2], ['one', 'two']],
           labels=[[0, 0, 1, 1], [0, 1, 0, 1]],
           names=['bar', None])
>>> idx.rename(['bar'])
ValueError: Length of names must match number of levels in MultiIndex.

See also
--------
set_names : able to set new names partially and by level

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	Errors in parameters section
		Parameter "name" description should start with capital letter
		Parameter "name" description should finish with "."
		Parameter "inplace" description should start with capital letter
		Parameter "inplace" description should finish with "."
	Examples do not pass tests

################################################################################
################################### Doctests ###################################
################################################################################

**********************************************************************
Line 29, in pandas.Index.rename
Failed example:
    idx.rename(['bar'])
Exception raised:
    Traceback (most recent call last):
      File "/home/mk/anaconda3/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest pandas.Index.rename[5]>", line 1, in <module>
        idx.rename(['bar'])
      File "/home/mk/documents/dev/pandas-mk/pandas/core/indexes/base.py", line 1341, in set_names
        idx._set_names(names, level=level)
      File "/home/mk/documents/dev/pandas-mk/pandas/core/indexes/multi.py", line 672, in _set_names
        raise ValueError('Length of names must match number of levels in '
    ValueError: Length of names must match number of levels in MultiIndex.
```

Reply:
Can you fix the "Errors found:" ? (capitalization and punctuation)